phpunit.xml.dist

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <phpunit bootstrap="./vendor/autoload.php" colors="true">
+    <php>
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="999999" />
+    </php>
```

Comment:
Why? Since we want to fix them? :)

Reply:
you mean why I set the limit of `999999`? I think by default we should not make the testsuite fail in case there are deprecations. Would be pretty bad for contributors if CI is failing for some totally unrelated reason, or?